webfrontend/src/router/router.tsx

```diff
@@ -20,7 +19,7 @@ const router = createBrowserRouter([
             {
                 path: "/messages",
                 element: <Messages />
-            },
+            }
         ]
```

Comment:
The comma is left there so that changes can be attributed to the correct person when a new item is added on. Please leave the comma there.